cmd/main.go

```diff
@@ -362,7 +364,15 @@ func main() { //nolint:funlen,maintidx,gocyclo
 		os.Exit(1)
 	}
 
-	webhook.Init(mgr)
+	// Register all webhooks using the shared helper
+	if err := shared.RegisterAllWebhooks(
```

Comment:
this is not working if you use `make run-nowebhook` since the related function are then not known, so it would fail with:

```
github.com/opendatahub-io/opendatahub-operator/v2/internal/webhook/datasciencecluster: build constraints exclude all Go files in /home/luca/work/dev/openshift-ai/opendatahub-operator/internal/webhook/datasciencecluster
github.com/opendatahub-io/opendatahub-operator/v2/internal/webhook/dscinitialization: build constraints exclude all Go files in /home/luca/work/dev/openshift-ai/opendatahub-operator/internal/webhook/dscinitialization
```